### PR TITLE
Fix CA1416 because we didn't set a min version

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

CA1416 - This call site is reachable on: 'Windows' 10.0.17763.0 and later. 'XXX' is only supported on: 'Windows' 10.0.18362.0 and later.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <Company>Microsoft</Company>
     <Product>Microsoft MAUI</Product>
     <ProduceReferenceAssembly Condition="'$(UsingMicrosoftNETSdk)' == 'True' AND '$(Configuration)' == 'Debug'">True</ProduceReferenceAssembly>
+    <TargetPlatformMinVersion Condition="'$(TargetPlatformMinVersion)' == '' and $(TargetFramework.Contains('-windows'))">10.0.17763.0</TargetPlatformMinVersion>
```
Comment (mattleibow) [inline Directory.Build.props:20]: This is the fix for the warning.

File: Directory.Build.props [hunk 2]
```diff
@@ -4,8 +4,8 @@
     <BuildForWinUI Condition="'$(SolutionFileName)' == 'Microsoft.Maui.WinUI.sln'">true</BuildForWinUI>
     <BuildForNet6 Condition="'$(SolutionFileName)' == 'Microsoft.Maui-net6.sln' or '$(BuildForWinUI)' == 'true'">true</BuildForNet6>
     <MauiPlatforms>net6.0-ios;net6.0-maccatalyst;net6.0-android</MauiPlatforms>
-    <MauiPlatforms Condition="'$(Packing)' == 'true'">$(MauiPlatforms);net6.0-windows10.0.19041.0</MauiPlatforms>
-    <MauiPlatforms Condition="'$(BuildForWinUI)' == 'true'">net6.0-windows10.0.19041.0</MauiPlatforms>
+    <MauiPlatforms Condition="'$(Packing)' == 'true'">$(MauiPlatforms);net6.0-windows10.0.18362.0</MauiPlatforms>
+    <MauiPlatforms Condition="'$(BuildForWinUI)' == 'true'">net6.0-windows10.0.18362.0</MauiPlatforms>
```
Comment (mattleibow) [inline Directory.Build.props:8]: We can use the same version that WinUI does.

File: src/Compatibility/Core/src/Compatibility-net6.csproj [hunk 3]
```diff
@@ -95,8 +94,8 @@
     <ProjectReference Include="..\..\..\Controls\src\Core\Controls.Core-net6.csproj" />
   </ItemGroup>
   <ItemGroup Condition="$([MSBuild]::IsOSPlatform('windows'))">
-    <None Include="bin\$(Configuration)\net6.0-windows10.0.19041.0\Microsoft.Maui.Controls.Compatibility.pri" Visible="False" Pack="True" PackagePath="runtimes\net6.0-windows10.0.19041.0\native" />  
-    <None Include="bin\$(Configuration)\net6.0-windows10.0.19041.0\Microsoft.Maui.Controls.Compatibility\**\*.xbf" Visible="False" Pack="True" PackagePath="lib\net6.0-windows10.0.19041\Microsoft.Maui.Controls.Compatibility;runtimes\net6.0-windows10.0.19041.0\native\Microsoft.Maui.Controls.Compatibility" />
+    <None Include="bin\$(Configuration)\$(TargetFramework)\Microsoft.Maui.Controls.Compatibility.pri" Visible="False" Pack="True" PackagePath="runtimes\$(TargetFramework)\native" />  
+    <None Include="bin\$(Configuration)\$(TargetFramework)\Microsoft.Maui.Controls.Compatibility\**\*.xbf" Visible="False" Pack="True" PackagePath="lib\$(TargetFramework)\Microsoft.Maui.Controls.Compatibility;runtimes\$(TargetFramework)\native\Microsoft.Maui.Controls.Compatibility" />
```
Comment (mattleibow) [inline src/Compatibility/Core/src/Compatibility-net6.csproj:98]: Using the variable here just in case we update and we forget.

**Review by jsuarezruiz — APPROVED** [review]

**Review by Redth — APPROVED** [review]

**Review by PureWeen — CHANGES_REQUESTED** [review]
Nugets look like they don't work

image

I had the folders a wee bit different depending on where they got put

```
PackagePath="lib\net6.0-windows10.0.19041\Microsoft.Maui.Controls.Compatibility;runtimes\net6.0-windows10.0.19041.0\native\Microsoft.Maui.Controls.Compatibility"
```

don't remember if that was a factor

**Review by PureWeen — APPROVED** [review]

**Pull request merged**